app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %> 
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true %> 
+  </div>
+
+    <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo %> 
+  </div>
```

Comment:
- [ ] In order to be consistent with the layout of the wireframe, please introduce a padding around the form.
- [ ] Also, notice that in the wireframe, there are no labels. Instead, set the placeholders for each input element so that you have a design that looks more like the wireframe design.
The above requested changes are project requirements 👇 
![image](https://github.com/Outkarm/blog-app/assets/46839548/44fbf389-0318-4e6d-b459-e8dcc2b57dd3)


## Wireframe
![image](https://github.com/Outkarm/blog-app/assets/46839548/71eeaf60-268c-40b9-a478-3f5238aeb5fb)